Physicist/Physicist/Controls/PhysicistSoundControllerException.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Physicist.Controls
```

Comment:
Not needed and not any different than Exception. Hack to get around CodeAnalysis. Used to throw exception in SoundController. ContentControl should be changed to throw an exception instead for asset not found or return a default sound, similar to the 'missing texture'.